lib/commands/addOn.js

```diff
@@ -102,79 +110,86 @@ addOn.chooseRegionPrompt = function (addon, callback) {
       maximum: addon.regions.length
     }
   ], function (err, result) {
-      if (err) {
-        return error.handlePromptError(err, callback);
-      }
+    if (err) {
+      return Errors.handlePromptError(err, callback);
+    }
 
-      modulus.io.print(util.format('Select %s\n', addon.regions[result.region - 1].data));
-      callback(null, addon.regions[result.region - 1]);
-    });
+    print(Util.format('Select %s\n', addon.regions[result.region - 1].data));
+    callback(null, addon.regions[result.region - 1]);
+  });
 };
 
-//-----------------------------------------------------------------------------
+// ----------------------------------------------------------------------------
 // Prints a list of Add-Ons
-//-----------------------------------------------------------------------------
-addOn.printList = function(projectName, addons) {
-  modulus.io.print('Add-Ons provisioned for ' + projectName.verbose);
+// ----------------------------------------------------------------------------
+addOn.printList = function (projectName, addons) {
+  var addonConfig,configItem, i;

```

Comment:
You need a space between the first two items. And can you put `i` first? At first I didn't see it...would be more obvious at the front.